docs/scripting/directives/script_call.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: "#Xs.user.script(...)"
+---
+
+Call another user's script.
```

Comment:
```suggestion
Call a script from within another script.
```
Should be clear this can also be another script on the same user, or even a trust script.